### PR TITLE
Create 2019-05-02-ux-guidelines-for-modules-developers

### PR DESCRIPTION
**Pull request by ttoine**

### CODE REVIEW

File: news/_posts/2019-05-02-ux-guidelines-for-modules-developers [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
```
Comment (eternoendless) [inline news/_posts/2019-05-02-ux-guidelines-for-modules-developers]: The file is missing the .md extension

**Review by eternoendless — COMMENTED** [review]
I cannot comment on the PDF but it uses the word "datas" which is incorrect in English

File: news/_posts/2019-05-02-ux-guidelines-for-modules-developers (hunk 1 above)
Comment (ttoine) [reply news/_posts/2019-05-02-ux-guidelines-for-modules-developers]: done, thanks

**Comment by ttoine** [comment]
@eternoendless unfortunately I am not sure we can edit the PDF anymore...

**Pull request merged**